.dockerignore

```diff
@@ -0,0 +1,5 @@
+api/tests/
+sentinel/tests/
+admin/tests/
+webapp/tests/
+tests/
```

Comment:
As I understand it, .dockerignore supports glob syntax, where we could codify the "tests" ignore with a single line like `**/tests`. Can we do that instead of keeping a specific list here?